src/test/java/test/inject/NativeInjectionTest.java

```diff
@@ -20,14 +20,15 @@ public void testBeforeSuiteInjection(Class clazz, String methodName, String expe
 
     @DataProvider
     public Object[][] getTestData() {
-        String variant1="Can inject only one of <ITestContext, XmlTest> into a %s annotated ";
-        String variant2="Can inject only one of <ITestContext, XmlTest, Method, Object[]> into a %s annotated ";
+        String variant1="Can inject only one of <ITestContext, XmlTest> into a @%s annotated ";
+        String variant2="Can inject only one of <ITestContext, XmlTest, Method, Object[]> into a @%s annotated ";
```

Comment:
variant2 is no more tested. Could you find an appropriate use case or remove the variant?